resources/themes/cura-light/icons/default/CheckBoxBorder.svg

```diff
@@ -0,0 +1,4 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
This file isn't used (and I also don't think it's needed, since it's pretty easy to create this in QML without using an SVG)